spec/integration/domchange_spec.rb

```diff
@@ -3,8 +3,7 @@
   scenario 'the DOM is ready' do
     visit '/domchange'
 
-    expect(page.driver.console_messages.map { |m| m[:message] })
-      .to eq(['uic:domchange was fired'])
+    expect(JS_LOGGER.string).to eq("uic:domchange was fired\n")
```

Comment:
I think `visit` waits for the page to load. And at that point, the event should have been fired already. This doesn't change the existing behaviour and I don't think we've had issues with this before. WDYT?